app/models/MetadataDefinition.scala

```diff
@@ -95,7 +95,12 @@ object MetadataDefinition {
         Json.parse("""
           {"label":"GeoJSON",
             "uri":"http://geojson.org/geojson-spec.html",
-            "type":"wkt"}""")
+            "type":"wkt"}"""),
+        Json.parse("""
+          {"label":"Vega5",
```

Comment:
can we make the label "Vega5-spec" instead of "Vega5"

Reply:
Done.